### PR TITLE
Update license headers to pmmp.io domain

### LINKED ISSUE #3106: Outdated website link
As noted in the title, Pocketmine still uses deprecated website links (http://www.pocketmine.net/) for their License instead of new one (https://pmmp.io/).

You can find this outdated link in all pocketmine classes at line 17.

Please also apologize, I did not find the type of problem between them because it's not a bug, it's not an API change request, nor a vulnerability, nor any of these issues templates that you have provided.

It's not a major change request, it does not affect anything, so I understand if this issue get closed.
Thank you!

### PR DESCRIPTION
**Pull request by buchwasa**
## Introduction
This pull requests reflects the domain change from pocketmine.net to pmmp.io

### Relevant issues
Closes #3106

### API changes
None

### Behavioural changes
None

## Backwards compatibility
Fully backwards compatible

## Follow-up
I don't have anything to add.
Edit: this will need to be reflected in other branches and PRs with new classes, that'll be fun
## Tests
The server still runs, so we good

### CODE REVIEW

**Comment by dktapps** [comment]
To paraphrase my comments on Discord, the reason this change has never been applied by the team is because, despite its apparent simplicity (a quick find+replace) it has a bunch of downstream effects which are not apparent to whoever wants the changes.

- It dirties the commit history: Every php file in the repo will be marked by this commit.
- It creates headaches merging branches: Integrating these changes into `master` will require a non-insignificant amount of work to resolve delete/modify inconsistencies and also to update headers for new files on that branch too.
- Literally nobody uses these links as a reference anyway, so this change is just a bunch of pain to satisfy someone else's OCD.
- pocketmine.net points to the PMMP documentation anyway.

TL;DR: This is more painful than it looks, and is all pain & no gain.

**Comment by dktapps** [comment]
Closing this with reasons mentioned above.
